projects/prysm-bn-validator-api.md

```diff
@@ -0,0 +1,66 @@
+# Prsym Beacon Node - Validator API
+
+Implementing the standard Ethereum Beacon API for validator clients in the Prysm beacon node
+
+## Motivation
+
+In production, the Prysm beacon node exposes a GRPC API for consumption by the Prysm validator.
+
+As a result of the standard Ethereum [spec](https://ethereum.github.io/beacon-APIs/#/Validator), this implementation is incompatible with other validator clients.
+
+Implementing the standard API will allow different validators to be run with the Prysm beacon node, furthering client diversity.
+
+## Project description
+
+An HTTP implementation of the Beacon API for validators currently exists in the Prysm beacon node.
+
+However this is a wrapper over a GRPC implementation, exposed using a fork of grpc-gateway. This is not ideal for several reasons, outlined in more detail in [prysm/#11580](https://github.com/prysmaticlabs/prysm/issues/11580).
+
+It also requires auditing & benchmarking to ensure optimal performance.
+
+## Specification
+
+A specification that actively tracks this project is available on [prysm/#12635](https://github.com/prysmaticlabs/prysm/issues/12635).
+
+Beyond this, technical details for the various endpoints are out of the scope of this document.
+
+## Roadmap
+
+As of August 15, HTTP handlers have been implemented for 6 of the 27 endpoints.
+
+The majority of this work has been contributed by the Prysm team, while I am currently working on my second endpoint.
```

Comment:
might be worth mentioning or linking which endpoints you did and plan to

Reply:
linked.

also, removed the exact number because i discovered some more endpoints that have been migrated but aren't tracked in the issue yet.